src/store/sagas/movies.js

```diff
@@ -47,7 +51,11 @@ function* loadMovies(action) {
     });
     yield put({
       type: GET_MOVIES_SUCCESS,
-      payload: { totalPages, movies },
+      payload: {
+        totalPages,
+        movies,
+        ids: getIds(movies),
```

Comment:
I think it's better to get this ids inside of reducer like we did it with normalize data. Place `getIds` helper func near `normalizeData` and use it inside reducer.
Your approach not completely correct because you did data transformations in two different places (1 - saga, 1 - reducer). Let's do all such transformations in reducer (like we did it with normalization).